checks/fuzzing.go

```diff
@@ -86,5 +86,5 @@ func Fuzzing(c *checker.CheckRequest) checker.CheckResult {
 			"project is fuzzed in OSS-Fuzz")
 	}
 
-	return checker.CreateMinScoreResult(CheckFuzzing, "project is not fuzzed")
+	return checker.CreateInconclusiveResult(CheckFuzzing, "could not determine if project is fuzzed")
```

Comment:
Let's make the message explicit - `project not fuzzed through any platforms recognizable by Scorecard: oss-fuzz, CFLite. File a support request [here](https://github.com/ossf/scorecard/issues/897) if you would like support for other fuzzing platforms.`. Can even keep this as a failing case instead of inconclusive.

Reply:
If we want to have a generic message `File a support request`, I think we should do this in a another PR and for all checks. The details has the details on which platforms are not found

Reply:
mhh, the details don't have this but it should. Ill update this PR to fix tat too